layers/meta-balena-beaglebone/recipes-support/resin-init/resin-init-flasher.bbappend

```diff
@@ -3,9 +3,17 @@ INTERNAL_DEVICE_KERNEL:beaglebone = "mmcblk1"
 # Except on the beaglebone-ai64
 INTERNAL_DEVICE_KERNEL:beaglebone-ai64 = "mmcblk0"
 
+# Because eMMC enumeration on the beagpleplay is different from other Beaglebone devices (?)
+INTERNAL_DEVICE_KERNEL:beagleplay = "mmcblk0"
+INTERNAL_DEVICE_KERNEL:beagleplay-k3r5 = "mmcblk0"
```

Comment:
We do not care about the k3r5 machine so we can remove this

Reply:
OK.

Reply:
Done.